pymatgen/io/vasp/MatPESStaticSet.yaml

```diff
@@ -0,0 +1,160 @@
+# Default VASP settings for static calculations in the MatPES Project
+# using the Regularized-Restored Strongly Constrained and Appropriately
+# Normed functional (PBE, PBE+U or r2SCAN).
+INCAR:
+  ALGO: ALL
+  EDIFF: 1.e-05
+  EDIFFG: -0.02
+  ENAUG: 1360
+  ENCUT: 680
+  IBRION: 2
+  ISIF: 3
+  ISMEAR: 0 # change from 2 to 0, included to have some reasonable default
+  ISPIN: 2
+  KSPACING: 0.22 # included to have some reasonable default
+  LAECHG: True
+  LASPH: True
+  LCHARG: True
+  LELF: False # LELF = True restricts calculation to KPAR = 1
+  LMIXTAU: True
+  LORBIT: 11
+  LREAL: Auto
+  LVTOT: True
+  LWAVE: False
+  METAGGA: PBE # change R2SCAN to PBE as default
```

Comment:
I don't think METAGGA=PBE is actually a valid value (see the VASP wiki). This should not be set in the YAML.

Reply:
I just saw on the VASP wiki that there was a GGA:PE. Can I put this in the INCAR part in the YAML? Then I can change the value of GGA and METAGGA for different functionals.

Reply:
Yes, that would be fine.